examples/csharp/azure/Main.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Constructs;
+using Hashicorp.Cdktf;
+
+
+namespace MyCompany.MyApp
+{
+    class MyApp : TerraformStack
+    {
+        public MyApp(Construct scope, string id) : base(scope, id)
+        {
+            // define resources here
```

Comment:
I'm guessing this is just in-progress still, but definitely should have some resources in it.